src/psyclone/domain/lfric/lfric_extract_driver_creator.py

```diff
@@ -273,6 +274,7 @@ def _flatten_reference(self, old_reference, symbol_table,
 
         :raises InternalError: if the old_reference is not a \
             :py:class:`psyclone.`StructureReference
+        raises GenerationError: if an array of structures is used
```

Comment:
Missing opening colon.

Reply:
Indeed. And the previous line had totally broken `StructureReference` specification :)